lib/components/atoms/Blockquote/tests/Blockquote.test.js

```diff
@@ -9,15 +9,15 @@ describe('<Blockquote />', () => {
   beforeEach(() => {
     BlockquoteComponent = shallow(<BlockquoteVanilla {...defaultBlockquote} />);
   });
-  it('check quoteProp value passed in', () => {
-    expect(BlockquoteComponent.props().quoteProp).not.toBeNull();
-  });
   it('should render blockquote', () => {
     expect(BlockquoteComponent.find('blockquote')).toBeTruthy();
   });
   it('should render content', () => {
     expect(BlockquoteComponent.find('p')).toBeTruthy();
   });
+  it.only('check quoteProp value passed in', () => {
```

Comment:
Who will check for author value? and default value of the same?

Reply:
author default value checked.